templates/cli/lib/utils.js.twig

```diff
@@ -14,6 +18,60 @@ function getAllFiles(folder) {
     return files;
 }
 
+function showConsoleLink(serviceName, action, id = '') {
+    let resource = '';
+    let service = '';
+
+    switch (serviceName) {
+        case "account":
+            service = 'account';
+            break;
+        case "databases":
+            resource = 'database';
+            service = 'databases';
+            break;
+        case "functions":
+            resource = 'function';
+            service = 'functions';
+            break;
+        case "projects":
+            service = `project-${id}`;
+            id = '';
+            break;
+        case "teams":
+            resource = 'team';
+            service = 'auth/teams';
+            break;
+
+        case "users":
+            resource = 'user';
+            service = 'auth';
+            break;
+        default:
+            return;
+    }
+
+    const baseUrl = globalConfig.getEndpoint().replace('/v1', '');
+
+    const end = action === 'get' ? (id ? `/${resource}-${id}` : `/${resource}`) : '';
+    const projectId = localConfig.getProject().projectId;
+    const middle = resource !== '' ? `/project-${projectId}` : '';
+    const url = `${baseUrl}/console${middle}/${service}${end}`
+
+    const start = (process.platform == 'darwin' ? 'open' : process.platform == 'win32' ? 'start' : 'xdg-open');
+    const key = (process.platform == 'darwin' ? 'Return' : 'Enter');
+
+    success(`\n  ${url}\n  Press <${key}> to open URL in your default browser, exising in 3 seconds`);
+    setTimeout(() => process.exit(0), 3000);
```

Comment:
Not sure if I like the timer. I would probably just show URL, and give hint they can use tag `--open` to open it in browser automatically.